package.json

```diff
@@ -184,4 +184,4 @@
     "node-fetch",
     "ua-parser-js"
   ]
-}
+}
```

Comment:
revert

Reply:
can't understand what you are trying to point out.